### DISCUSSION TITLE
Chromium works on Arch Linux

### DISCUSSION THREAD

**Discussion by hello-smile6** [copy/v86#692] [General]
Screenshot:
![Screenshot 2022-06-16 9 22 58 AM](https://user-images.githubusercontent.com/73048226/174119130-58686c1e-c75a-46ff-94b9-ba4f87d69d72.png)
It takes a lot of work to configure, but I'm going to share everything needed to get Chromium to start.

**Comment by ethanaobrien** [comment]
(I think this is more of a discussion)

**Comment by hello-smile6** [comment]
> (I think this is more of a discussion)

Could you convert it?

**Reply by Darin755** [reply]
The only way that I know of to convert this is to delete it and then reopen it

**Comment by Darin755** [comment]
@hello-smile6 you might want to try a web browser that is a little lighter on resources

**Reply by hello-smile6** [reply]
@Darin755 Good idea.

**Comment by srikantpatnaik** [comment]
Great!
I tried Slax 9.9.1. It has chromium pre-installed in ~250MB ISO. 
![image](https://user-images.githubusercontent.com/703521/174931363-04d02f80-d4cd-4bfe-bc17-904d78431f20.png)
![image](https://user-images.githubusercontent.com/703521/174931452-3b2c6700-00ec-4cde-8ab9-baf64d126ef3.png)

**Reply by Darin755** [reply]
does v86 actually work inside itself?

**Reply by hello-smile6** [reply]
@srikantpatnaik Just wondering, _how_ good exactly is your CPU? I've never gotten Slax to _boot_ that fast in v86, let alone open a browser.

**Reply by srikantpatnaik** [reply]
@Darin755 yes, nested emulation worked. I used Buildroot and it took around 30 mins to get me to the prompt. But soon, it got unresponsive. 
@hello-smile6 My CPU `11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz`

**Reply by hello-smile6** [reply]
> @Darin755 yes, nested emulation worked. I used Buildroot and it took around 30 mins to get me to the prompt. But soon, it got unresponsive. @hello-smile6 My CPU `11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz`

Wow, could you record a video and share it? That must be incredible. Guess I just only use slow computers.

**Reply by srikantpatnaik** [reply]
@hello-smile6 
I can't enter text inside the nested emulator. 

https://user-images.githubusercontent.com/703521/175102699-6616a29c-768f-48ec-808a-b026e490dbfe.mp4
